.github/workflows/main.yml

```diff
@@ -0,0 +1,44 @@
+name: Build Tests
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+      - release-*
```

Comment:
@boaks has suggested not to use the release-branches for tinydtls. As there are currently no releases and no release branches, I suggest removing this entry to avoid confusion or erroneous assumptions.

Reply:
Removed and code pushed with cmake.  clang to follow.

Reply:
Thanks. I would expect that clang can just be added to the compile matrix (line 23).